frontend/src/components/App/Home/RecentClusters.tsx

```diff
@@ -86,6 +93,33 @@ export default function RecentClusters(props: RecentClustersProps) {
     });
   }
 
+  /**
+   * Callback for when different clusters are toggled.
+   *
+   * @param event - The event that triggered the toggle.
+   * @param clusters - The clusters that are toggled.
+   */
+  function onClustersToggled(event: React.MouseEvent<HTMLElement>, clusters: Cluster[]) {
```

Comment:
this function could be inlined (minor nit)

Reply:
done